apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -1372,7 +1385,7 @@ export async function truncateAllTables(conn: Conn, schema: string) {
 export async function dropElement (conn: Conn, elementName: string, typeOfElement: DatabaseElement, schema: string = 'public'): Promise<void> {
   await runWithConnection(conn, async (connection) => {
     const connClient = { connection };
-    const sql = `DROP ${PD.wrapLiteral(typeOfElement)} ${wrapIdentifier(schema)}.${wrapIdentifier(elementName)}`
+    const sql = `DROP ${PD.wrapLiteral(DatabaseElement[typeOfElement])} ${wrapIdentifier(schema)}.${wrapIdentifier(elementName)}`
```

Comment:
Forgive my ignorance here - how is this change different in function to the prior version?

Reply:
it works like a key-value this way, it didn't make any difference since `typeOfElement` were same as values (one word), but with materialized view `typeOfElement` is `materialized-view`, but value that has to be filled in here is `MATERIALIZED VIEW`, I thought this was the easiest way how fix it.

Reply:
Ok, sounds good. Looking forward to the other minor changes then will merge :+1: